src/android/java/com/mattrayner/vuforia/app/ImageTargets.java

```diff
@@ -57,40 +57,41 @@
 public class ImageTargets extends Activity implements ApplicationControl
 {
     private static final String LOGTAG = "ImageTargets";
-    
+    private static final String FILE_PROTOCOLE_FIX = "file://";
+
```

Comment:
Yes of course, I think I wanted to use "FILE_PROTOCOL_PREFIX" instead,
that's just a mistake.
By the way, I didn't use any constant for the iOS part, so that's a bit
ugly, but I can still fix it if you want.

2016-06-08 17:45 GMT+02:00 daniel seán griffin <notifications@github.com>:

> In src/android/java/com/mattrayner/vuforia/app/ImageTargets.java
> <https://github.com/mattrayner/cordova-plugin-vuforia/pull/34#discussion_r66281694>
> :
>
> > @@ -57,40 +57,41 @@
> >  public class ImageTargets extends Activity implements ApplicationControl
> >  {
> >      private static final String LOGTAG = "ImageTargets";
> > -
> > +    private static final String FILE_PROTOCOLE_FIX = "file://";
> > +
>
> Could we rename this to be FILE_PROTOCOL?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mattrayner/cordova-plugin-vuforia/pull/34/files/fb1db5ca4d12f49248c58b54552260e70bdc5b4a#r66281694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANWwn2LUR13wvBxvlXgGnhy6orMw8cNyks5qJuODgaJpZM4Iw9l0>
> .
>